astronomer/providers/snowflake/sensors/snowflake.py

```diff
@@ -90,20 +90,13 @@ def execute(self, context: Context) -> None:
                 method_name=self.execute_complete.__name__,
             )
 
-    def execute_complete(
-        self,
-        context: Context,
-        event: dict[str, str | list[str]] | None = None,
-    ) -> Any:
+    def execute_complete(self, context: Context, event: dict[str, Any] | None = None) -> Any:
```

Comment:
Why do we change the type annotation from `Any` to `list[str]`? Is it no longer accurate?

Reply:
I took a quick look and look like it yield 
```
yield TriggerEvent(
                            {
                                "status": "success",
                                "message": "Found expected markers.",
                            }
                        )
```
so was not correct?

Reply:
got it. then should it be `dict[str, str]` instead?